# migrate command in south cause this error:table "model_name" already exist
python manage.py migrate DrHub --fake
